### PR TITLE
Mary Denise Robertson homework assignment 1

### PR DESCRIPTION
**Pull request by marydeniserobertson**
I included my reflection in the assignment but now I see this is the correct place for it:

Description of how I edited README: 
I went to the class page on github, made a fork to my account, cloned the fork to my Rstudio, read the markdown guide, wrote my intro, made two commits, and then generated this pull request.

Reflection on process:
Figuring out how to open the git file through Rstudio was tricky(the fork, clone part of the workflow). I originally thought we had to use the shell the way we did when first making repos and committing changes when testing out our software configuration. Once I found the step by step instructions for cloning it into Rstudio, it was pretty straightforward. R Markdown was very straightforward - the linked 'mastering markdown' tutorial on Github was really useful. Remembering to commit, then pull, push was also a learning curve, but having the step by step checklist for the workflow is very helpful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# hw01
+# Homework assigment 1
 #Bio of Mary Denise Robertson
```
Comment (eggst) [inline README.md:2]: Prof Soltoff told me that putting a space between the hashtag and first letter makes it so the hashtag doesn't transfer over to the final view (I did the exact same thing).

**Comment by eggst** [comment]
**Evaluation**

Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | x |   |  
Coding strategy | x |   |  
Presentation | x |  |  
Achievement, creativity |  | x |  
Ease of access | x |   |  

Remarks:

- It's really cool that you're from South Africa! I see you've adjusted well to Chicago life but I guess that's to be expected after 9 years :) (I've only been here for about 3 weeks!)
- I felt like you could have given more substance and organization. You have the code down so I would be interested to know about you're study interests as an anthropologist (I am one too!) and definitely where those free milkshakes are at.

File: README.md (hunk 1 above)
Comment (marydeniserobertson) [reply README.md:2]: Thank you, that makes sense! Cool that another anthropologist is taking the class :-) Actually, the milkshakes are a dollar, not free, and they are at Pret in Reynolds - or at least they were.

**Comment by laurenguillot** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Your only minor mistake in your code was failing to put a space after # to create a header--a mistake it appears many others in our course (including myself), made as well. Although you did include the space in two of your headers (Homework assignment 1 and Time at UChicago)
* I like your layout, it looks like something you would include on a professional bio.
* Looking at your assignment, I realize we really need to be more mindful of minutiae like spaces
* I think many people in the class sought to complete everything the assignment asked of us, but did not go beyond the specific instructions. Programming is not a discipline that people typically think of as "creative" but we need to challenge that perspective in this course and strive to be creative and even innovative with our work.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
-# hw01
-Repository for homework 01
+# Homework assigment 1
+#Bio of Mary Denise Robertson
+I was born in South Africa, and came to Chicago 9 years ago to study a PhD in Anthropology. I like Chicago except the winters are *very* cold. And **very** long. 
+
+## Time at UChicago
+I like UChicago. My three favorite things are 
+
+* Naps in Harper Library
```
Comment (laurenguillot) [inline README.md:8]: Great job with the list! Your issue with spacing appears to be confined to the use of the # in making headers.

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |           |      x       |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* You don't need to commit all files such as .gitignore or .Rproj files. Just commit those files which are necessary.
* Issue with headers but glad to see that you resolved it.

**Pull request closed without merging**